wings/1.0/installing.md

```diff
@@ -18,8 +18,7 @@ only the versions listed below.
 
 | Operating System | Version |     Supported      | Notes                                                       |
 |------------------|---------|:------------------:|-------------------------------------------------------------|
-| **Ubuntu**       | 18.04   | :white_check_mark: | Documentation written assuming Ubuntu 18.04 as the base OS. |
-|                  | 20.04   | :white_check_mark: |                                                             |
+| **Ubuntu**       | 20.04   | :white_check_mark: | Documentation written assuming Ubuntu 20.04 as the base OS. |
 |                  | 22.04   | :white_check_mark: |                                                             |
```

Comment:
```
| **Ubuntu**       | 20.04   | :white_check_mark: |                                                             |
|                  | 22.04   | :white_check_mark: | Documentation written assuming Ubuntu 20.04 as the base OS. |
```

Reply:
?

Reply:
wait my bad, i thought it should be at the bottom as every other note